config/rbac/role.yaml

```diff
@@ -110,6 +137,7 @@ rules:
   resources:
   - datavolumes
   verbs:
+  - '*'
```

Comment:
the `*` should not be here

Reply:
@ksimon1 Can we remove it in another PR?

Reply:
yes

Reply:
@ksimon1 I see that it caused by:
```
// +kubebuilder:rbac:groups=cdi.kubevirt.io,resources=datavolumes,verbs=*

```
Do you know what verbs we should specify then?

Reply:
Everything is taken from https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/tasks

Reply:
Thanks. Verbs will be tracked in [1].

[1] https://issues.redhat.com/browse/CNV-31843